compiler/src/dmd/dsymbolsem.d

```diff
@@ -5831,6 +5832,365 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
     }
 }
 
+/*
+Adds dsym as a member of scope sds.
+
+Params:
+    dsym = dsymbol to inserted
+    sc = scope where the dsymbol is declared
+    sds = ScopeDsymbol where dsym is inserted
+*/
+extern(C++) void addMember(Dsymbol dsym, Scope* sc, ScopeDsymbol sds)
```

Comment:
This function is unused by LDC: @ibuclaw is it used by GDC? If not this should be `extern(D)`

Reply:
This can be done in a subsequent PR.